src/shared/use-user-info/use-user-info.js

```diff
@@ -8,8 +8,25 @@ const queryOpts = {
     staleTime: 24 * 60 * 60 * 1000,
 }
 
+const parseLocale = (unparsedLocale) => {
+    if (typeof unparsedLocale !== 'string') {
+        return 'en'
+    }
+
+    // while script is possible in DHIS2 locales, it is not supported in underlying multi-calendar-dates logic
+    const [lng, region] = unparsedLocale?.replace(/_/g, '-').split('-')
```

Comment:
the language codes returned from the backend should use `_` instead of `-`, but I'm doing a replace before split in case any start being returned in the ISO format